packages/web/docs/src/app/product-updates/page.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: Product Updates
+description: The most recent developments from GraphQL Hive.
+---
+
+import { ProductUpdatesPage } from '../../components/product-updates'
+
+<div className="[&>h1]:!text-left [&>h1]:!mb-0 [&>p]:!mt-0">
+  # Product Updates
```

Comment:
This is wild :D but it works

Reply:
in mdx you can have markdown syntax in spagetty code of html elements e.g.

```mdx
<div>
  <div>
    <div>
      ## Hello world
   </div>
  </div>
</div>
```

Reply:
but not `<div>## Hello World</div>` <- will be string `## Hello world`